### PR TITLE
Aliases Update for Minecraft 1.17

### PR DESCRIPTION
**Pull request by APickledWalrus**
This PR aims to add aliases for then new blocks and items added into Minecraft 1.17

I used the wiki along with a lovely document Shane gave me for reference (thanks!) to try and get everything.
However, it is possible I still missed some things so if anyone wants to double check I would appreciate it.
It is also possible I made some mistakes, as there were a lot of new blocks to cover.

### CODE REVIEW

File: decoration.sk [hunk 1]
```diff
@@ -651,7 +651,67 @@ nether update:
...
+	{waterloggable} {orientable} amethyst cluster¦s = minecraft:amethyst_cluster
+	
+	azalea¦s = minecraft:azalea
+	flowering azalea¦s = minecraft:flowering_azalea
+	{decayable} azalea lea(f|ves) = minecraft:azalea_leaves
+	{decayable} flowering azalea lea(f|ves) = minecraft:flowering_azalea_leaves
+	[any] lea(f|ves) = any leaves, azalea leaves, flowering azalea leaves
+	
+	candle = minecraft:candle
+	{colored} candle = -candle
```
Comment (TheBentoBox) [inline decoration.sk]: These two candle lines should have the `¦s` plurality marker.

File: decoration.sk [hunk 2]
```diff
@@ -651,7 +651,67 @@ nether update:
...
+		middle = -[thickness=middle]
+		tip = -[thickness=tip]
+		tip merge = -[thickness=tip_merge]
+	{pointed dripstone direction}:
+		{default} = -
+		(floor [attached]|upward facing) = -[vertical_direction=down]
+		(ceiling [attached]|downward facing) = -[vertical_direction=up]
+	{waterloggable} {pointed dripstone direction} pointed dripstone {pointed dripstone thickess} = minecraft:pointed_dripstone
+	
+	spore blossom = minecraft:spore_blossom
```
Comment (TheBentoBox) [inline decoration.sk]: This should have the `¦s` plurality marker.

File: misc.sk [hunk 3]
```diff
@@ -300,10 +300,31 @@ nether update:
...
 
 	(snout|piglin) banner pattern¦s = minecraft:piglin_banner_pattern
 	[any] banner pattern¦s = flower charge banner pattern, creeper charge banner pattern, skull charge banner pattern, thing banner pattern, globe banner pattern, snout banner pattern
+
+caves and cliffs update part 1:
+	minecraft version = 1.17 or newer
+	
+	glow ink sac¦s = minecraft:glow_ink_sac
+	
+	amethyst shard = minecraft:amethyst_shard
```
Comment (TheBentoBox) [inline misc.sk]: This should have the ¦s plurality marker.

File: other.sk [hunk 4]
```diff
@@ -237,3 +237,47 @@ nether update:
 		north-up = -[orientation=north_up]
 		south-up = -[orientation=south_up]
 	{jigsaw-orientation} jigsaw block¦s = minecraft:jigsaw
+
+caves and cliffs update part 1:
+	minecraft version = 1.17 or newer
+	
+	candle = minecraft:candle_cake
+	{colored} candle = -candle_cake
```
Comment (TheBentoBox) [inline other.sk]: This and the above line are missing the `cake` suffix, should be `candle cake¦s` and `{colored} candle cake¦s`

**Review by TheBentoBox — COMMENTED** [review]
Good stuff man, a few small issues

**Review by TheBentoBox — APPROVED** [review]

**Comment by APickledWalrus** [comment]
Need to make some changes as aliases will not currently load on 1.17 correctly

**Comment by APickledWalrus** [comment]
🎆 With this third pass, the aliases all successfully load on 1.17 (tested with debug mode on and stuff) 🥳 

I think this PR is good to go now - but one final look from someone to make sure the name patterns for everything is good wouldn't hurt.

**Review by ShaneBeee — APPROVED** [review]
Looks good to me

**Pull request merged**